storage/ndb/src/kernel/vm/mt_thr_config.cpp

```diff
@@ -426,9 +426,6 @@ THRConfigApplier::do_bind(NdbThread* thread,
     case T_QUERY:
```

Comment:
it seems that T_QUERY is not removed completely:
   1     58  include/mgmcommon/thr_config.hpp <<GLOBAL>>
             T_QUERY = 9,
   2     58  src/common/mgmcommon/thr_config.cpp <<GLOBAL>>
             { "query", THRConfig::T_QUERY},
   3     83  src/common/mgmcommon/thr_config.cpp <<GLOBAL>>
             { THRConfig::T_QUERY, 0, MAX_NDBMT_QUERY_WORKERS, true, true, 0 },
   4   1356  src/common/mgmcommon/thr_config.cpp <<handle_spec>>
             type == T_QUERY ||
   5   1762  src/kernel/vm/Configuration.cpp <<setupConfiguration>>
             m_thr_config.getThreadCount(THRConfig::T_QUERY);
   6    426  src/kernel/vm/mt_thr_config.cpp <<do_bind>>
             case T_QUERY:

Reply:
ok, cleaned up and removed all references to Query threads, now only Query workers left